utility/src/test/java/org/oppia/android/util/utility/NetworkConnectionTestUtil.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.android.util.utility
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import org.robolectric.Shadows
+import org.robolectric.shadows.ShadowNetworkInfo
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Test utility to modify [ShadowNetworkInfo] during the test. */
+@Singleton
```

Comment:
rather than creating it singleton and injecting, I don't see any reason to make this class singleton. 
What other way we can do is, make `setNetworkInfo` static by putting it inside the companion object and access accordingly.

Reply:
Done.